securesystemslib/metadata.py

```diff
@@ -102,7 +102,7 @@ def pae(self) -> bytes:
             self.payload,
         )
 
-    def sign(self, signer: Signer) -> Signature:  # pragma: no cover
+    def sign(self, signer: Signer) -> Signature:
```

Comment:
Can you please move all changes for this file from this commit to the https://github.com/in-toto/securesystemslib/pull/2/commits/f1f84c3d168c523b7ea8be5057c1af2e4d832aa0 commit as these changes are not related to the actual purpose of this commit. 

Additionally, you add `pragma: no cover` there and delete it here. This way if you move these changes there it will be as if nothing has happened.

Reply:
I moved all the changes in correct commits.